src/mscorlib/src/System/AggregateException.cs

```diff
@@ -431,6 +432,30 @@ public AggregateException Flatten()
             return new AggregateException(Message, flattenedExceptions);
         }
 
+        /// <summary>Gets a message that describes the exception.</summary>
+        public override string Message
+        {
+            get
+            {
+                if (m_innerExceptions.Count == 0)
+                {
+                    return base.Message;
+                }
+
+                StringBuilder sb = StringBuilderCache.Acquire();
+                sb.Append(base.Message);
+                sb.Append(' ');
+                for (int i = 0; i < m_innerExceptions.Count; i++)
+                {
+                    sb.Append('(');
+                    sb.Append(m_innerExceptions[i].Message);
+                    sb.Append(") ");
+                }
+                sb.Length -= 1;
```

Comment:
@stephentoub  curious, was there a reason not to just put the space inside the for loop:
```cs
StringBuilder sb = StringBuilderCache.Acquire();
sb.Append(base.Message);
for (int i = 0; i < m_innerExceptions.Count; i++)
{
    sb.Append(" (");
    sb.Append(m_innerExceptions[i].Message);
    sb.Append(')');
}
```

Reply:
That'd probably be better.